Dockerfile

```diff
@@ -5,15 +5,18 @@
 FROM node:14-buster as base
```

Comment:
Something crashes the build pipelines, probably these changes. Check build output on this PR

Reply:
Fixed!